sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/BlobClientJavaDocCodeSnippets.java

```diff
@@ -36,7 +36,7 @@
  */
 @SuppressWarnings("unused")
 public class BlobClientJavaDocCodeSnippets {
-    private BlobClient client = JavaDocCodeSnippetsHelpers.getBlobClient("blobName");
+    private BlobClient client = new BlobClientBuilder().blobName("blobName").buildClient();
```

Comment:
Should configure what is actually needed in the builders for them to function properly.